plugins/cisco_umbrella_destinations/plugin.spec.yaml

```diff
@@ -0,0 +1,405 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: cisco_umbrella_destinations
+title: Cisco Umbrella Destinations
+version: 1.0.0
+description: Cisco Umbrella Destinations allows users to manage, block, and allow network destinations based on policies within their organization
+vendor: rapid7
+support: rapid7
+status: []
+resources:
```

Comment:
Please add at least one supported version of Cisco Umbrella that this plugin was tested with, changelog here: https://support.umbrella.com/hc/en-us/sections/206928207-Release-Notes-for-Umbrella-software